src/Viewer/components/Monaco/MonacoInstance.scss

```diff
@@ -4,3 +4,9 @@
 // .monaco-editor-overlaymessage {
 //  display: none !important;
 // }
+
+.monaco-container {
+    display: flex;
```

Comment:
The editor doesn't seem to resize with the browser anymore. @junhaoliao mentioned we likely need [`automaticLayout`](https://microsoft.github.io/monaco-editor/typedoc/interfaces/editor.IStandaloneEditorConstructionOptions.html#automaticLayout) set in the editor's options. With that set, we also don't need `display: flex` here.